test/input-states-mocha.js

```diff
@@ -0,0 +1,333 @@
+/**
+ * File: input-state-mocha.js
+ * Unit tests for the input-state APIs defined in src/input-state.js
+ */
+
+var chai = require('chai');
+var expect = chai.expect;
+var should = chai.should();
+
+var _ = require('underscore');
+var componentFactory = require('../src/noflo-component-factory');
+var test = require('./common-test');
+var inputStates = require('../src/input-states');
+var createLm = require('../src/create-lm');
+
+describe('input-states', function() {
+    var node, network;
+    beforeEach(function(){
+        var getComponent = componentFactory({
+            inPorts:{
+                input:{},
+                input1:{},
+                input2:{},
+                input3:{},
+                inputa:{
+                    addressable: true
```

Comment:
Given we are using isMulti, perhaps one of these should use that setting instead of addressable?